suite-common/wallet-core/src/accounts/accountsReducer.ts

```diff
@@ -89,6 +90,12 @@ export const selectAccounts = (state: AccountsRootState) => state.wallet.account
 export const selectAccount = (state: AccountsRootState, accountKey: string) =>
     state.wallet.accounts.find(account => account.key === accountKey);
 
+export const selectDeviceNetworkAccounts =
+    (deviceState: string, networkSymbol: NetworkSymbol) => (state: AccountsRootState) =>
```

Comment:
What is wrong with `selectAccounts`? @Nodonisko

Reply:
I meant `selectAccout` (without "s"). It should use createSelector that will use `selectAccounts` and input api of createSelector

Reply:
If you look at the docs, you'll see that they add parameters using other selectors. And since this one just gets a parameter and returns part of state, I didn't find it necessary to use createSelector.

Reply:
But you should use other selectors if possible, in this case there should be used `selectAccounts` instead of `state.wallet.accounts`. This may seems unecessary but it's actually very useful because it minimalizing amount of changes during refactorings.